src/lib/loaders/__tests__/batchLoader.test.ts

```diff
@@ -143,37 +160,94 @@ describe("groupKeys", () => {
       },
     ]
     expect(groupKeys(keys)).toEqual([
-      {
-        id: ["a", "b"],
-        foo: "bar",
-        size: 2,
-      },
+      ["foo=bar"],
+      [
+        [
+          {
+            id: "a",
+            foo: "bar",
+          },
+
+          {
+            id: "b",
+            foo: "bar",
+          },
+        ],
+      ],
     ])
   })
 
   it("should separate keys with different parameters", () => {
     const keys = [
-      "a",
+      { id: "a" },
       { id: "b", foo: "bar" },
       { id: "c", foo: "bar", boo: "baz" },
     ]
 
     expect(groupKeys(keys)).toEqual([
-      {
-        id: ["a"],
-        size: 1,
-      },
-      {
-        id: ["b"],
-        foo: "bar",
-        size: 1,
-      },
-      {
-        id: ["c"],
-        foo: "bar",
-        boo: "baz",
-        size: 1,
-      },
+      ["", "foo=bar", "boo=baz&foo=bar"],
+      [
+        [
+          {
+            id: "a",
+          },
+        ],
+        [
+          {
+            id: "b",
+            foo: "bar",
+          },
+        ],
+        [
+          {
+            id: "c",
+            foo: "bar",
+            boo: "baz",
+          },
+        ],
+      ],
     ])
   })
 })
+
+describe("cacheKeyFn", () => {
+  const { cacheKeyFn } = require("../batchLoader")
+  it("should treat strings and ids no params the same", () => {
```

Comment:
Not sure what “ids no params” means here

Reply:
Ahh, so the object actually represents query parameters for a gravity call. So I'd kind of continued with that language.